js/src/wp-seo-replacevar-plugin.js

```diff
@@ -529,5 +529,14 @@ var ReplaceVar = require( "./values/replaceVar" );
 		return parentText;
 	};
 
+	/**
+	 * STATIC VARIABLES
+	 */
+
+	/**
```

Comment:
Quick note: double asterisks should only be used for docblocks. Instead, multi-line comments should have just one asterisk: `/*`